ci-operator/step-registry/cucushift/hypershift-extended/external-oidc/health-check/cucushift-hypershift-extended-external-oidc-health-check-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
From an overall perspective, I do not wish to add a new health check. Instead, this part of the code should be merged into `cucushift-hypershift-extended-external-health-check`

Reply:
@LiangquanLi930 This health check is for ext-oidc clusters only.

Reply:
yes, I know, we should enable a step to support more scenarios, rather than adding more steps; it is just a check.

Reply:
1) Handling all scenarios in a monolithic step makes maintenance challenging.
2) As far as I can tell, this ext-oidc health check is harmless and easy to maintain.

Reply:
@YuLi517 Nope. I don't think so.